astrokat/test/test_offline_observe.py

```diff
@@ -35,11 +35,20 @@ def test_targets_sim(self):
 
         # get result and make sure everything ran properly
         result = LoggedTelescope.user_logger_stream.getvalue()
+        self.assertIn("target0_radec at azel (179.9, 30.6) deg", result)
```

Comment:
When running Aqf (see Anton's comment in the description on how to here - https://skaafrica.atlassian.net/browse/MT-2603). We get this failure from tailing the SB progress output.
```bash
FAIL: Test targets sim.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/mock/mock.py", line 1330, in patched
    return func(*args, **keywargs)
  File "/home/kat/svn/astrokat/astrokat/test/test_offline_observe.py", line 38, in test_targets_sim
    self.assertIn("target0_radec at azel (179.9, 30.6) deg", result)
AssertionError: 'target0_radec at azel (179.9, 30.6) deg' not found
``

Reply:
after some investigation, see - http://10.8.67.104:8081/tailtask/20210913-0010/progress

these `assertIn` calls are checking for things that don't exist in the SB tasklog, thought the `unittests` have them. The Aqf tests are failing at the SB level. We will have to make both tests to pass.